src/assets/css/utilities/_text.css

```diff
@@ -10,21 +10,18 @@
 
 /* BIMDATA FONT SIZE */
 .bimdata-h1 {
-  font-size: 34px;
-}
-.bimdata-h2 {
   font-size: 30px;
 }
-.bimdata-h3 {
+.bimdata-h2 {
   font-size: 26px;
 }
-.bimdata-h4 {
+.bimdata-h3 {
   font-size: 24px;
 }
-.bimdata-h5 {
+.bimdata-h4 {
   font-size: 20px;
 }
-.bimdata-h6 {
+.bimdata-h5 {
   font-size: 16px;
 }
```

Comment:
So we don't have `h6` heading anymore ?

Reply:
We don't use h6, on the other hand if we used h6  it will make a title the size of a text (14px) and bold